templates/vue/src/components/SettingsModal.vue

```diff
@@ -109,6 +109,45 @@
               </b-col>
             </b-row>
           </b-form-group>
+          <b-form-group
+            label="World map"
+            description="Replace Tapestry with a new authoring experience over a map of the Earth"
```

Comment:
This should read "Replace Tapestry with a map of Earth with placeholders for each node."